qiskit/algorithms/optimizers/__init__.py

```diff
@@ -52,6 +64,8 @@
    L_BFGS_B
    GSLS
    GradientDescent
+   GradientDescentState
+   CMAES
```

Comment:
At one point the list was in alphabetic order so it was easier to find something. I guess it still is mostly, though L_BFGS_B seems the worst offender - it might be better to move CMAES to try to have things in order.